packages/neon/neon_dashboard/lib/src/pages/main.dart

```diff
@@ -156,6 +160,86 @@ class DashboardMainPage extends StatelessWidget {
               return const SizedBox.shrink();
             },
           ),
+          StreamBuilder(
+            stream: weatherStatusBloc.isSupported,
+            builder: (context, weatherStatusSupportedSnapshot) => ResultBuilder.behaviorSubject(
+              subject: weatherStatusBloc.forecasts,
+              builder: (context, forecastsResult) {
+                if (weatherStatusSupportedSnapshot.data ?? false) {
```

Comment:
```suggestion
            if (weatherStatusSupportedSnapshot.data == null || !weatherStatusSupportedSnapshot.data!) {
                  return const SizedBox.shrink();
                }
```

I find early returns are easier to read.